### ISSUE TITLE
Enhancement:  Suppress Switch Group until 2 or more rules

### ISSUE THREAD

**Issue by ccarns** [zebzhao/Angular-QueryBuilder#100]
My user base will mostly being using only 1 rule most of the time.  In this case the querySwitchGroup (AND or OR option) does not change the final query results so is just extra screen noise for basic users (I know, I know maybe I should use  a different component for this use case but ideally I am looking for basic and adv functionality in one component).  I was thinking about supressing the querySwitchGroup options until a second rule is added.  It seems fairly easy to do but checking if anybody else was looking for this as an enhancement or maybe point out issues with it I may have not considered.

**Comment by MolallaComm** [comment]
yes - i was thinking the same thing - this would be useful as alot of the time, you'll only need a filter on one column
